openedx/features/course_card/views.py

```diff
@@ -77,6 +89,8 @@ def get_course_with_link_and_start_date(course, course_rerun_object, request):
     if current_class:
         current_class_start_date = get_course_open_date(current_class)
 
+    course.specialization = any(course_program)
```

Comment:
Once we will remove this specialization flag from CurseOverview then we can remove this change from here. Even with current implementation instead of doing this here in this function we can do this at `71` instead of passing course_prorgram to this method.